OpenEdXMobile/src/test/java/org/edx/mobile/view/CourseUnitNavigationActivityTest.java

```diff
@@ -446,7 +443,10 @@ public void testUnitFragmentCreation() {
         unitList.add(othersModel);
         classesList.add(CourseUnitMobileNotSupportedFragment.class);
 
-        CourseUnitPagerAdapter adapter = new CourseUnitPagerAdapter(fragmentManager, config,
+        ActivityController<? extends CourseUnitNavigationActivity> controller =
```

Comment:
this is mentioned as ignored, do we still need to update this one?
also, only tests are passing in this class

Reply:
we have o change the code cuz, the definition of the method/construct has been changed.